pkg/features/list.go

```diff
@@ -4,47 +4,59 @@ package features
 
 	// NetworkDetectionBaselineSimulation enables new features related to the baseline simulation part of the network detection experience.
-	NetworkDetectionBaselineSimulation = registerFeature("Enable network detection baseline simulation", "ROX_NETWORK_DETECTION_BASELINE_SIMULATION", true)
+	NetworkDetectionBaselineSimulation = registerUnchangeableFeature("Enable network detection baseline simulation", "ROX_NETWORK_DETECTION_BASELINE_SIMULATION", true)
 
 	// IntegrationsAsConfig enables loading integrations from config
-	IntegrationsAsConfig = registerFeature("Enable loading integrations from config", "ROX_INTEGRATIONS_AS_CONFIG", false)
+	IntegrationsAsConfig = registerUnchangeableFeature("Enable loading integrations from config", "ROX_INTEGRATIONS_AS_CONFIG", false)
 
 	// QuayRobotAccounts enables Robot accounts as credentials in Quay Image Integration.
-	QuayRobotAccounts = registerFeature("Enable Robot accounts in Quay Image Integration", "ROX_QUAY_ROBOT_ACCOUNTS", true)
+	QuayRobotAccounts = registerUnchangeableFeature("Enable Robot accounts in Quay Image Integration", "ROX_QUAY_ROBOT_ACCOUNTS", true)
 
 	// RoxctlNetpolGenerate enables 'roxctl netpol generate' command which integrates with NP-Guard
-	RoxctlNetpolGenerate = registerFeature("Enable 'roxctl netpol generate' command", "ROX_ROXCTL_NETPOL_GENERATE", true)
+	RoxctlNetpolGenerate = registerUnchangeableFeature("Enable 'roxctl netpol generate' command", "ROX_ROXCTL_NETPOL_GENERATE", true)
 
 	// RoxSyslogExtraFields enables user to add additional key value pairs in syslog alert notification in cef format.
-	RoxSyslogExtraFields = registerFeature("Enable extra fields for syslog integration", "ROX_SYSLOG_EXTRA_FIELDS", true)
+	RoxSyslogExtraFields = registerUnchangeableFeature("Enable extra fields for syslog integration", "ROX_SYSLOG_EXTRA_FIELDS", true)
 
 	// SourcedAutogeneratedIntegrations enables adding a "source" to autogenerated integrations.
-	SourcedAutogeneratedIntegrations = registerFeature("Enable autogenerated integrations with cluster/namespace/secret source", "ROX_SOURCED_AUTOGENERATED_INTEGRATIONS", false)
+	SourcedAutogeneratedIntegrations = registerUnchangeableFeature("Enable autogenerated integrations with cluster/namespace/secret source", "ROX_SOURCED_AUTOGENERATED_INTEGRATIONS", false)
 
 	// VulnMgmtWorkloadCVEs enables APIs and UI pages for the VM Workload CVE enhancements
-	VulnMgmtWorkloadCVEs = registerFeature("Vuln Mgmt Workload CVEs", "ROX_VULN_MGMT_WORKLOAD_CVES", true)
+	VulnMgmtWorkloadCVEs = registerUnchangeableFeature("Vuln Mgmt Workload CVEs", "ROX_VULN_MGMT_WORKLOAD_CVES", true)
 
 	// PostgresBlobStore enables the creation of the Postgres Blob Store
-	PostgresBlobStore = registerFeature("Postgres Blob Store", "ROX_POSTGRES_BLOB_STORE", false)
+	PostgresBlobStore = registerUnchangeableFeature("Postgres Blob Store", "ROX_POSTGRES_BLOB_STORE", false)
```

Comment:
The default value seems wrong. The blob store is created and exists since 4.2.

Reply:
Apparently this is not used anywhere. Not sure if it was ever used actually